tests/integration/routes/test_view_preview_questions.py

```diff
@@ -0,0 +1,22 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestViewSubmissionResponsePDF(IntegrationTestCase):
+    def test_download(self):
+        super().setUp()
+
+        # Given I launch and complete a questionnaire that has view-submitted-response enabled and has expired
```

Comment:
Is this correct that a questionnaire with preview enabled needs view-submitted-response enabled?

Reply:
Description is correct now, previous one was from existing test i used as base.